Tweaking of Pharmaceuticals text for resubmission (#978)
greenelab/covid19-review: content/20.pharmaceuticals.md

### Abstract

After emerging in China in late 2019, the novel _Severe acute respiratory syndrome-like coronavirus 2_ (SARS-CoV-2) spread worldwide and as of early 2021 continues to significantly impact most countries.
Only a small number of coronaviruses are known to infect humans, and only two cause infections similar in severity to SARS-CoV-2: _Severe acute respiratory syndrome-related coronavirus_, a closely related species of SARS-CoV-2 that emerged in 2002, and _Middle East respiratory syndrome-related coronavirus_, which emerged in 2012.

> Unlike in the current pandemic, both of these previous epidemics were controlled fairly rapidly through public health measures.

Reviewer: ```suggestion
Unlike in the current pandemic, both of these previous epidemics were controlled relatively rapidly through public health measures.
```

Due to the continued threat of the virus and the severity of the disease, the identification and development of therapeutic interventions have emerged as significant international priorities.
Prior developments during other recent outbreaks of emerging diseases, especially those caused by human coronaviruses (HCoV), have guided biomedical research into the behavior and treatment of this novel coronavirus infection.
However, previous emerging HCoV-related disease threats were controlled much more quickly than SARS-CoV-2 through public health efforts (Figure @fig:csse-deaths).
The scale of the COVID-19 pandemic has made the repurposing and development of pharmaceuticals more urgent than in previous coronavirus epidemics.

#### Lessons from Prior HCoV Outbreaks

![
**Cumulative global incidence of COVID-19 and SARS.**
As of {{csse_date_pretty}}, {{csse_cases}} COVID-19 cases and {{csse_deaths}} COVID-19 deaths had been reported worldwide since January 22, 2020 (Figure @fig:csse-deaths).

> In contrast, a total of 8,432 cases and 813 deaths were reported for SARS from March 17 to July 11, 2003.

Reviewer: Is in contrast required here, when lines 43 and 44 are deleted?

Author: ```suggestion
A total of 8,432 cases and 813 deaths were reported for SARS from March 17 to July 11, 2003.
```

Therefore, SARS-CoV-2 emerged under very different circumstances than SARS-CoV-1 in terms of scientific knowledge about HCoVs and the tools available to characterize them.

Despite the apparent advantages for responding to SARS-CoV-2 infections, COVID-19 has caused many orders of magnitude more deaths than SARS did (Figure @fig:csse-deaths).
The SARS outbreak was officially determined to be under control in July 2003, with the success credited to infection management practices such as mask wearing [@doi:10.1038/nrmicro.2016.81].
_Middle East respiratory syndrome-related coronavirus_ (MERS-CoV) is still circulating and remains a concern; although the fatality rate is very high at almost 35%, the disease is much less easily transmitted, as its R~0~ has been estimated to be 1 [@doi:10.1093/ajcp/aqaa029].
The low R~0~ in combination with public health practices allowed for its spread to be contained [@doi:10.1093/ajcp/aqaa029].
Neither of these trajectories are comparable to that of SARS-CoV-2, which remains a serious threat worldwide over a year and a half after the first cases of COVID-19 emerged (Figure @fig:csse-deaths).

#### Potential Approaches to the Treatment of COVID-19

> Therapeutic interventions can utilize two approaches: they can either mitigate the effects of an infection that harm an infected person, orthey can hinder the spread of infection within a host by disrupting the viral life cycle.

Reviewer 2: ```suggestion
Therapeutic interventions can utilize two approaches: they can either mitigate the effects of an infection that harm an infected person, or they can hinder the spread of infection within a host by disrupting the viral life cycle.
```

These on-the-ground treatment strategies could later be analyzed retrospectively in observational studies or investigated in an interventional paradigm through randomized control studies.
Several notable cases involved the use of small-molecule drugs, which are synthesized compounds of low molecular weight, typically less than 1 kilodalton (kDa) [@url:https://www.nuventra.com/resources/blog/small-molecules-versus-biologics/].
Small-molecule pharmaceutical agents have been a backbone of drug development since the discovery of penicillin in the early twentieth century [@doi:10.1126/science.287.5460.1960].
It and other antibiotics have long been among the best known applications of small molecules to therapeutics, but biotechnological developments such as the prediction of protein-protein interactions have facilitated advances in precise targeting of specific structures using small molecules [@doi:10.1126/science.287.5460.1960].
Small molecule drugs today encompass a wide range of therapeutics beyond antibiotics, including antivirals, protein inhibitors, and many broad-spectrum pharmaceuticals.

Many treatments considered for COVID-19 have relied on a broad-spectrum approach.

> Such treatments do not specifically target a virus or particular host receptor, but rather induce broad shifts in host biology that are hypothesized to be potential inhibitors of the virus.

Reviewer 2: Is this specifically in the antiviral space (as written, I think it is)? It feels like the immunomodulatory treatments (especially steroids) are the quintessential broad-spectrum treatment.

Author: Good point! Something definitely got lost here in the rearranging.
```suggestion
In addition to broad-spectrum treatments like dexamethasone, antivirals offer an alternative approach.
These treatments do not specifically target a virus or particular host receptor, but rather induce broad shifts in host biology that are hypothesized to be potential inhibitors of the virus.
```

Reviewer 2: 👍

Only interventional trials are analyzed in the figures depicting status, phase, and intervention.
Of the interventional trials, {{ebm_trials_results}} trials had reported results as of {{ebm_date_pretty}}.
Recruitment status and trial phase are shown only for interventional trials in which the status or phase is recorded.
Common interventions refers to interventions used in at least ten trials.
Combinations of interventions, such as hydroxychloroquine with azithromycin, are tallied separately from the individual interventions.
]({{ebm_trials_figure}} "COVID-19 clinical trials"){#fig:ebm-trials secno=1}

The purpose of this review is to provide an evolving resource tracking the status of efforts to repurpose and develop drugs for the treatment of COVID-19.
We highlight four strategies that provide different paradigms for the identification of potential pharmaceutical treatments.
As results become available from additional clinical trials, we will continue to update this manuscript to keep pace with the current understanding of which therapeutics may be effective against SARS-CoV-2 or COVID-19.

### Repurposing Drugs for Symptom Management

> A variety of symptom profiles with a range of severity are associated with COVID-19.

Reviewer: Can probably cite the pathogenesis manuscript there

### Considerations in Balancing Different Approaches

The range of approaches described here offer a range of advanctages and limitations in responding to a novel viral threat and build on existing bodies of knowledge in different ways.
A host-oriented, symptom management approach allows clinicians to apply experience with related diseases or related symptoms to organize a rapid response aimed at saving the lives of patients already infected with a new disease.
Oftentimes, the pharmaceutical agents that are applied are small-molecule, broad-spectrum pharmaceuticals that are widely available and affordable to produce, and they may already be available for other purposes, allowing clinicians to administer them to patients quickly either with an EUA or off-label.
However, the fact that the targets of these agents are non-specific means that the mechanism of action can appear to be relevant to COVID-19 without a therapeutic or prophylactic effect being observed in clinical trials.

> Over the last year, dexamethasone has emerged as the strongest treatment against severe COVID-19, but other clinically suggested compounds such as HCQ/CQ and ARBs/ACEis have not borne out early hope in their mechanistic relevance.

Reviewer 2: Small molecule treatment? Would vaccines be included here?

Author: I thought the vaccines are biologics -- but I was only in the pharmacology department for eight months, so will definitely defer to you 😆  Regardless, we should probably mention vaccines in this discussion...
```suggestion
Over the last year, dexamethasone has emerged as the strongest treatment against severe COVID-19, but other clinically suggested compounds such as HCQ/CQ and ARBs/ACEis have not borne out early hope in their mechanistic relevance.
A number of vaccine development programs have also met huge success using a range of strategies [@individual-vaccines].
```

Reviewer 2: It's up to you - I just wasn't sure if we were grouping or distinguishing between them. I'd consider this optional :)

Author: Thanks @cgreene! I'm curious what @RLordan thinks!

Reviewer: I'm just as stumped haha, but if this helps the Center for Biologics Evaluation and Research regulates childhood vaccines in the US

Author: Thank you @RLordan and @cgreene!
